test.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: apps/v1
```

Comment:
Do you still need this or was this for local testing?

Reply:
thanks! I've removed this.